### PR TITLE
Update functionality of code examples

### PR DESCRIPTION
**Pull request by SenseException**

### CODE REVIEW

**Review by greg0ire — DISMISSED** [review]

**Comment by greg0ire** [comment]
We might even want to use type declarations here, WDYT?

**Comment by SenseException** [comment]
@greg0ire I've added types to the methods. Do you think I should add those to properties too? It looks weitd that `addEntry` has a return type too.

File: docs/en/cookbook/aggregate-fields.rst [hunk 1]
```diff
@@ -32,30 +32,39 @@ Our entities look like:
 .. code-block:: php
 
     <?php
+
     namespace Bank\Entities;
+
+    use use Doctrine\ORM\Mapping as ORM;
```
Comment (greg0ire) [inline docs/en/cookbook/aggregate-fields.rst:38]:
```suggestion
    use Doctrine\ORM\Mapping as ORM;
```

File: docs/en/cookbook/aggregate-fields.rst [hunk 2]
```diff
@@ -175,12 +190,13 @@ relation with this method:
     <?php
     class Account
     {
-        public function addEntry($amount)
+        public function addEntry(int $amount): Entry
```
Comment (greg0ire) [inline docs/en/cookbook/aggregate-fields.rst:193]: That does look weird, not sure where it comes from.

**Review by greg0ire — COMMENTED** [review]
> Do you think I should add those to properties too?

Yes, I think we should promote modern code.

File: docs/en/cookbook/aggregate-fields.rst (hunk 2 above)
Comment (SenseException) [reply docs/en/cookbook/aggregate-fields.rst]: I don't see any usage of the return value so I removed it.

**Comment by greg0ire** [comment]
> SenseException requested a review from greg0ire 2 hours ago


What about adding types to properties?

**Comment by SenseException** [comment]
I can add those too. PHP 7.3 is close to the end of bugfix support and adding property types would have been part of another PR, but sure: why not start now?

**Review by greg0ire — APPROVED** [review]

File: docs/en/cookbook/aggregate-fields.rst [hunk 3]
```diff
@@ -64,31 +73,35 @@ Our entities look like:
...
-        public function __construct($account, $amount)
+        public function __construct(Account $account, int $amount)
         {
             $this->account = $account;
             $this->amount = $amount;
             // more stuff here, from/to whom, stated reason, execution date and such
         }
     
-        public function getAmount()
+        public function getAmount(): Amount
```
Comment (svycka) [inline docs/en/cookbook/aggregate-fields.rst:104]: the return value should be int instead of Amount

**Pull request merged**